### PR TITLE
Prufer Algorithm implemented in Python

### PR DESCRIPTION
**Pull request by Riyazul555**
@iuliagroza do check this out too.....closes PR #39

### CODE REVIEW

File: Prufer Decoding Algorithm/Prufer.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (iuliagroza) [inline Prufer Decoding Algorithm/Prufer.py:1]: Delete extra line

File: Prufer Decoding Algorithm/Prufer.py
Comment (iuliagroza) [inline Prufer Decoding Algorithm/Prufer.py:1]: Let's rename to "Prufer Decoding/prufer_decoding.py".

**Review by iuliagroza — COMMENTED** [review]
Code looking great!

**Comment by Riyazul555** [comment]
Done @iuliagroza ......

**Review by iuliagroza — APPROVED** [review]

**Comment by iuliagroza** [comment]
@Riyazul555 Thanks for the awesome contributions! :)

**Pull request merged**